packages/@aws-cdk/aws-cloudfront/lib/experimental/edge-function.ts

```diff
@@ -148,8 +148,8 @@ export class EdgeFunction extends Resource implements lambda.IVersion {
 
   /** Create a support stack and function in us-east-1, and a SSM reader in-region */
   private createCrossRegionFunction(id: string, props: EdgeFunctionProps): FunctionConfig {
-    const parameterNamePrefix = 'EdgeFunctionArn';
-    const parameterName = `${parameterNamePrefix}${id}`;
+    const parameterNamePrefix = '/EdgeFunctionArn';
```

Comment:
maybe `/cdk/EdgeFunctionArn` here?

Reply:
Sure, I can see that. I'd be happy with this as-is, but open to the `/cdk` prefix. Do you want to update?